core/server/middleware/validation/blog-icon.js

```diff
@@ -61,7 +61,7 @@ module.exports = function blogIcon() {
 
         // CASE: file should not be larger than 100kb
         if (!validIconSize(req.file.size)) {
-            return next(new errors.RequestEntityTooLargeError({message: i18n.t('errors.api.icons.fileSizeTooLarge', {extensions: iconExtensions})}));
+            return next(new errors.ValidationError({message: i18n.t('errors.api.icons.invalidFile', {extensions: iconExtensions})}));
```

Comment:
I changed this to be a `ValidationError`, because the client would show a different error message to the user in this case. But we want to show the user only one error message for all cases.